nixos/modules/services/web-apps/pict-rs.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, pkgs, config, ... }:
+with lib;
+let
+  cfg = config.services.pict-rs;
+in
+{
+  options.services.pict-rs = {
+    enable = mkEnableOption "pict-rs server";
```

Comment:
Bonus points of you mention something about throwing this begging a reverse proxy and include upstream documentation on how to do so